docs/rackhd/tasks.rst

```diff
@@ -148,145 +151,88 @@ The primary difference between the *Install CoreOS* task and the *Install Ubuntu
 
-**Relation between Job & Schema & Task**
+**Schema Classification**
 
-- A Task Definition conforms to only one Task Schema, but different Task Definition can conform to the same Task Schema; For example, all obm-control tasks share the same obm-control schema.
-- A Task Schema only describes one Job, but a Job may be described by multiple Task Schemas. For example, all of install-centos, install-ubuntu, install-esxi schemas are to describe the same install-os job.
-- A Task Schema can include other schemas for the sake of schema reuse and minimizing duplication. For example, all task schemas include a common options schema.
+There are totally 3 kinds of options schema: Common options schema, BaseTask options schema and Task options schema.
 
-Below is a diagram shows the relation:
+- The common options schema is to describe all those common options that shared by all tasks, such as `_taskTimeout`, the common options schema is defined in the file 'https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/common-task-options.json'. User doesn't have to explicitly define the common schema in Task or BaseTask definition, it is defaultly enabled for every task.
 
-.. image:: /_static/task_schema_job_relation.png
-  :align: center
-
-**Task Meta Schema**
-
-The Task Meta Schema is the schema of Task Schema, it restricts the syntax of Task Schema.
-
-The name of Task Meta Schema is rackhd-task-schema.json_, any Task Schema should write against it by set the *$schema*:
-
-.. _rackhd-task-schema.json: https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/rackhd-task-schema.json
-
-.. code-block:: JSON
-
-    "$schema": "rackhd-task-schema.json"
-
-The meta schema is bases on the JSON-Schema_ draft-04 standard (http://json-schema.org/draft-04/schema), and it extends following keywords:
-
-============== ======= ========= =========================================================
-Keyword        Format  Flags     Description
-============== ======= ========= =========================================================
-describeJob    String  Required  The property is required to point to a job’s di injectable name
-copyright      String  Optional  This attribute shall contain the copyright notice for the schema
-readonly       Boolean Optional  This property shall designate a property to be readonly for user when set to true
-============== ======= ========= =========================================================
-
-**Define a Task Schema**
+- The schema in BaseTask is to describe the options of the corresponding job.
 
-You can follow below 3 steps to define a Task Schema:
+- The schema in Task definition is to describe the options of corresponding task. Since a task defintion will always link to a BaseTask, so the task's schema will automatically inherit the BaseTask's schema during validation. So in practice, usually the task schema only need to describe those options that not included in BaseTask.
 
```

Comment:
need => needs or to remove to
that not included => that are not included or remove that.